reconcile/user_validator.py

```diff
@@ -44,9 +46,11 @@ def validate_users_gpg_key(users):
 
 def run(dry_run):
     users = queries.get_users()
+    settings = queries.get_app_interface_settings()
+    smtp_client = SmtpClient(settings=settings)
 
     single_path_ok = validate_users_single_path(users)
-    gpg_ok = validate_users_gpg_key(users)
+    gpg_ok = validate_users_gpg_key(users, smtp_client)
```

Comment:
Pass the `org_username` directly, the gpg-handling methods have no business dealing with SMTP clients.